dtfs-central-api/src/services/state-machines/utilisation-report/event-handlers/report-uploaded/report-uploaded.event-handler.ts

```diff
@@ -34,6 +42,14 @@ export const handleUtilisationReportReportUploadedEvent = async (
   });
   await transactionEntityManager.save(UtilisationReportEntity, report);
 
+  const reportCsvDataFacilityIds = reportCsvData.map(({ 'ukef facility id': facilityId }) => facilityId);
+  const allReportCsvDataFacilityIdsExist = (
+    await Promise.all(reportCsvDataFacilityIds.map((facilityId) => transactionEntityManager.existsBy(FacilityUtilisationDataEntity, { id: facilityId })))
```

Comment:
Remove this check (is in another ticket and should check against the tfm collection)